website/templates/website/base_frame.html

```diff
@@ -28,4 +28,5 @@
 {% block footer %}{% endblock %}
 </body>
 {% block extra_js_bottom %}{% endblock %}
+{{ google_analytics|safe }}
```

Comment:
Out of curiosity, why did you choose to use a template parsing here with a context processor, instead of just hardcoding the setting here? For ease of changing it later on?